sites/fast-website/src/app/components/content-placement-container/content-placement-container.template.ts

```diff
@@ -20,23 +25,31 @@ export const ContentPlacementContainerTemplate = html<ContentPlacementContainer>
             <site-card-section>
                 ${repeat(
                     x => x.featureCardData,
-                    html`<site-feature-card>
-                        <div>${x => x.item}</div>
-                        <h4>${x => x.header}</h4>
-                        <p slot="body">${x => x.body}</p>
-                        <fast-anchor
-                            slot="footer"
-                            href=${x => x.githubLink}
-                            appearance="lightweight"
-                            >View Github</fast-anchor
-                        >
-                        <fast-anchor
-                            slot="footer"
-                            href=${x => x.documentationLink}
-                            appearance="lightweight"
-                            >Read Documentation</fast-anchor
-                        >
-                    </site-feature-card>`
+                    html`
+                        <site-feature-card>
+                            <div>
```

Comment:
There may be a better way to list these as an `<ol>` and utilize CSS numbering. I'll investigate.

Reply:
Thank you so much. I have updated it to use css numbering.